recipes/BEHST/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+mkdir $PREFIX/../results/
+
+mkdir $PREFIX/../temp/
+
+
+sh project.sh $PREFIX/../data/pressto_LUNG_enhancers.bed DEFAULT_EQ DEFAULT_ET
```

Comment:
Actually now that I read the source, this line is doing a functional test on example data. Is that correct? For bioconda tests we're only concerned with whether things made it onto the path correctly; we don't want to consume community resources on unit tests (see https://bioconda.github.io/guidelines.html#tests).

Also, we have limited space in the bioconda channel. Continuumio is already being generous with our quota, but we're close to the limit. Data should not be included in the package. If you want to provide a mechanism for getting example data for testing or for users to acquire, see for example [this recipe](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/chromhmm) or [this recipe](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/spectacle).